packages/expo-modules-autolinking/src/reactNativeConfig/reactNativeConfig.ts

```diff
@@ -30,7 +30,7 @@ export async function createReactNativeConfigAsync({
 }: RNConfigCommandOptions): Promise<RNConfigResult> {
   const projectConfig = await loadConfigAsync<RNConfigReactNativeProjectConfig>(projectRoot);
   const dependencyRoots = {
-    ...(await findDependencyRootsAsync(projectRoot, searchPaths)),
+    ...(await findDependencyRootsAsync(projectRoot, searchPaths, platform)),
     ...findProjectLocalDependencyRoots(projectConfig),
```

Comment:
```suggestion
    ...findProjectLocalDependencyRoots(projectConfig),
    ...(await maybeAddEdgeToEdgeDependencyAsync(projectRoot, platform),
```

leaning toward to add a new function to handle the case. also add unit test for the new function